docs/_docs/user-guide/golem.md

```diff
@@ -24,6 +24,126 @@ cargo build --release && \
     ../target/debug/golem ../../tests/golem_cli_test/tomes/hello_world.tome
 ```
 
+## Creating and testing tomes
+
+Golem is a great way to create and test tomes without deploying the entire Realm project.
```

Comment:
Golem is a great way to quickly develop and iterate tomes without needing to deploy an agent and wait for callbacks. It enables a more interactive experience for Eldritch, which helps tome developers move quickly.